snapcraft/internal/repo/snaps.py

```diff
@@ -94,6 +94,77 @@ def in_store(self):
                 self._is_in_store = False
         return self._is_in_store
 
+    def _get_installed_path(self):
+        return os.path.join("/snap", self.name, "current")
```

Comment:
Add a note mentioning that this requires building on a system that lays out snaps into /snap

Reply:
Are there other known paths?   Or a better way to query this...? I couldn't find anything obvious with a quick search.

Reply:
Another option would be to skip the install, and download the content snap and extract it (with some common code in SnapCache._get_snap_deb_arch).  Or install it as first resort, if we can't find it, fall back to downloading it?

Reply:
Went with common.get_base_path() for now as it's the same logic, will generalize the name later.

Reply:
Added commit to do rename.